nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs

```diff
@@ -795,7 +795,7 @@ internal static async Task<ImmutableArray<string>> GetTargetFrameworkValuesFromP
             );
             return (exitCode, stdOut, stdErr);
         });
-        ThrowOnUnauthenticatedFeed(stdOut);
+        ThrowOnError(stdOut);
```

Comment:
This function is a wrapper aroung `ThrowOnUnauthenticatedFeed` as well as a bunch of others, including `ThrowOnMissingFile`.